src/features/movies/MoviePage/Backdrop/index.js

```diff
@@ -1,36 +1,33 @@
 import {
-  BackdropContainer,
-  Details,
-  MaxRates,
-  Rates,
+  BlureFrames,
+  Container,
+  Content,
+  Rate,
   RatingContainer,
   StarIcon,
   Title,
+  Vote,
   Votes,
   Wrapper,
 } from "./styled";
-import { StyledBackdrop } from "./styled";
+import { BackdropImg } from "./styled";
 
 export const Backdrop = () => {
   return (
     <Wrapper>
-      <BackdropContainer>
-        <Details>
+      <Container>
+        <BlureFrames />
+        <BackdropImg src={"https://image.tmdb.org/t/p/w1280/bQXAqRx2Fgc46uCVWgoPz5L5Dtr.jpg"} alt="" />
+        <Content>
           <Title>Mulan long title</Title>
           <RatingContainer>
             <StarIcon />
-            <Rates>7,8</Rates>
-            <MaxRates>/ 10</MaxRates>
+            <Vote>10</Vote>
```

Comment:
tutaj już też będziemy przekazywać dane

Reply:
No racja, ale to lepiej dodac ten Backdrop i pozniej dodamy mu logike :)

Reply:
Na callu uzgodniliśmy, żebym wstawił go na sztywno ;D Ta gałąź jest wybita z maina o jeszcze nie pełnym API (czyli nie ma zamian od Szymona), więc nie chciałem kombinować np. z ręcznym kopiowaniem plików. Mogę podać mu wartości z API i liczyć na to że po mergeu skuma o co chodzi ;D